
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

upper_arm_controller: 
  type: position_controllers/JointTrajectoryController
  joints: 
    - swing_joint
    - boom_joint
    - arm_joint
    - bucket_joint
    - bucket_end_joint
  gains:
    swing_joint: { p: 100, i: 1,d: 1,i_clamp: 1}
    boom_joint: { p: 100, i: 1,d: 1,i_clamp: 1}
    arm_joint: { p: 100, i: 1,d: 1,i_clamp: 1}
    backet_joint: { p: 100, i: 1,d: 1,i_clamp: 1}
    backet_end_joint: { p: 100, i: 1,d: 1,i_clamp: 1}